src/workerd/api/worker-loader.c++

```diff
@@ -135,16 +135,24 @@ Worker::Script::Source WorkerLoader::extractSource(jsg::Lock& js, WorkerCode& co
   auto modules = KJ_MAP(entry, code.modules.fields) -> Worker::Script::Module {
     KJ_SWITCH_ONEOF(entry.value) {
       KJ_CASE_ONEOF(text, kj::String) {
+        if (entry.name.endsWith(".py"_kj)) {
```

Comment:
Hmm. If we're going to use file extensions to auto-detect types then I feel like we should also require that implicitly-typed JavaScript files end with `.js`, otherwise each time we add a language we're possibly breaking people if, like, they had JS files with a `.py` extension before.